ui/desktop/electron-app/config/forge.config.js

```diff
@@ -4,6 +4,13 @@ const { isMac } = require('../src/helpers/platform.js');
 const fs = require('fs');
 const path = require('path');
 
+// When SETUP_CLI=false we do not download the CLI. If there is no cli directory present
```

Comment:
@ZedLi 
Change the approach as per suggestion.

Now, if the cli directory exists, we will copy the cli in. If the directory does not exists we don't.

This translates the responsibility of having CLI or not to whoever runs the build process.

Changes on boundary-ui-releases to use this on tis way.

Reply:
I just realize, assuming we are performing 2 builds, once with CLI other without (reasons explained below) I think we want to differentiate the build with no CLI.
The binary current name is: Boundary.app
I do propose for no cli: Boundary-nocli.app

thoughts?

Why we need to perform 2 builds?

It was suggested to just perform one build, and then to provide no CLI just use the normal build and delete the CLI within Boundary.app/contents/... but after quick testing that is not feasible, or at least not as easy as we though. I will put the details of why is not within RFC within abandoned ideas.